### ISSUE TITLE
Empty text on OSD

### ISSUE THREAD

**Issue by VinceBT** [A-tG/VoicemeeterFancyOSD#26]
I'm using [v0.13.0.0](https://github.com/A-tG/VoicemeeterFancyOSD/releases/tag/v0.13.0.0) and I get an empty text when the OSD appears and it stays stuck in color red

![chrome_VW5zLvmwqG](https://user-images.githubusercontent.com/1362505/170053532-91457851-f580-43da-a265-773b6cf93c77.png)

**Comment by A-tG** [comment]
Try to reinstall program. Delete old files (except *config* folder) and download from Releases page. If it will not help something might be wrong with your Voicemeeter installation or vmrapi.

**Comment by VinceBT** [comment]
It seems to happen at the first initialization after my computer boots up. If I kill the program and relaunch it, I don't have the bug until the next reboot.

I've reinstalled, I'll tell you. I cannot say it worked because of course now I had to kill it and launch the new files so of course the bug has gone.

**Comment by A-tG** [comment]
Might be fixed in [new version](https://github.com/A-tG/VoicemeeterFancyOSD/releases/tag/v0.13.1.0). Try to change new option InitializationDelay to 5000 or 10000.

My only guess this bug was happening because of "early" connection to VoicemeeterAPI on System startup. With this option it will wait before loading and connecting to API.

**Comment by hallo545401** [comment]
First of all, amazing application. I also got this problem quite a few times now and I tried it with a delay of 10000. Weird thing is it only happens sometimes and not always. Restarting Voicemeeter or FancyOSD doesn't fix the problem for me. Its only on the VAIO3 Input for me.

**Comment by A-tG** [comment]
Try to enable Logger with new [v0.13.2.0](https://github.com/A-tG/VoicemeeterFancyOSD/releases/tag/v0.13.2.0) release.

This is how usual log file looks like when everything function properly.
```
25-05-2022 21:10:07.977 Reading config: OK
25-05-2022 21:10:07.980 Writing config...
25-05-2022 21:10:07.987 Writing config: OK
25-05-2022 21:10:07.989 Initializing VmrApi Client
25-05-2022 21:10:08.243 VmrApi Login result: 0
25-05-2022 21:10:08.279 VmrApi WaitForNewParams returned: 1
25-05-2022 21:10:08.547 VmrApi Client initialized
25-05-2022 21:10:08.547 Program initialized
```
But I suspect something wrong with Voicemeeter itself. I use Potato 3.0.2.1 32bit version and never had problems like that

**Comment by A-tG** [comment]
Nothing unusual in these logs. I've added more logging in 0.13.3.0 version maybe it will catch some internal .NET error

**Comment by VinceBT** [comment]
```
01-06-2022 13:06:21.136 Reading config: OK
01-06-2022 13:06:21.137 Writing config...
01-06-2022 13:06:21.142 Writing config: OK
01-06-2022 13:06:31.157 Initializing VmrApi Client
01-06-2022 13:06:32.004 VmrApi Login result: 0
01-06-2022 13:06:32.050 VmrApi WaitForNewParams returned: 1
01-06-2022 13:06:32.207 VmrApi Client initialized
01-06-2022 13:06:32.207 Program initialized
```
Still same issue
![image](https://user-images.githubusercontent.com/1362505/171822578-fd051ab5-59f7-4bf6-adf0-9ef84b3c5e54.png)![image](https://user-images.githubusercontent.com/1362505/171822505-c4f6058c-2fab-449b-be00-13204f323960.png)

**Comment by A-tG** [comment]
Do Faders (sliders) function properly apart from that the gain is not displayed? Looks like some .NET or WPF bug I have no idea how to fix that. Maybe try to install Windows update related to net framework, make sure that you are using 6.0.5 .NET Desktop Runtime (latest stable for the moment),

**Comment by hallo545401** [comment]
For me they do. It's only the vaio3 slider and it goes up and down normally. The color change and the value are not working.

**Comment by VinceBT** [comment]
I saw you made an update and I want to thank you for the quick response, I'll keep you updated if I encounter the bug again

**Comment by Glaived** [comment]
> Do Faders (sliders) function properly apart from that the gain is not displayed? Looks like this is some .NET or WPF bug, I have no idea how to fix that. Maybe try to install Windows update related to net framework, make sure that you are using 6.0.5 .NET Desktop Runtime (latest stable at the moment).

```
PS C:\Users\Glaived\Desktop> dotnet --list-runtimes
Microsoft.NETCore.App 3.1.25 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 6.0.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.WindowsDesktop.App 3.1.25 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
Microsoft.WindowsDesktop.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
Microsoft.WindowsDesktop.App 6.0.5 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

@A-tG I don't have a log to provide you, but I can inform you that I sometimes have this problem even though I'm up to date on version 6.0.5

**Comment by A-tG** [comment]
The only explanation that I came to: text in the thumb of the slider (that circle with the number) is Binded to Value of slider, but thumb is located in different UI Template than slider and I have to use little dirty hack to "find" and bind it to slider. Looks like sometimes it may fail, and Binding errors usually do not trow any exceptions, so no logs. I've [tried](https://github.com/A-tG/VoicemeeterFancyOSD/commit/a5519b2839bd1f72f1c41141edf56aed6bd40c0b) to make Binding more direct in [v0.13.3.1](https://github.com/A-tG/VoicemeeterFancyOSD/releases/tag/v0.13.3.1) so I hope it's fixed

**Comment by hallo545401** [comment]
I got the bug again with the same logs as I once sent above (same as Vince's).

**Comment by edfrenk** [comment]
Just wanted to add that I'm experiencing the issue on the current version as well. I'm using [this](https://www.turnup.tech/) to adjust my Voicemeeter sliders, and I'm getting no text and the slider stays red in the OSD. It's similar to the PCPanel that the other user mentioned. My logs seem fine as well? 

```27-08-2022 10:46:41.945 Reading config: OK
27-08-2022 10:46:41.946 Writing config...
27-08-2022 10:46:41.953 Writing config: OK
27-08-2022 10:46:51.958 Initializing VmrApi Client
27-08-2022 10:46:52.193 VmrApi Login result: 0
27-08-2022 10:46:52.253 VmrApi WaitForNewParams returned: 1
27-08-2022 10:46:52.556 VmrApi Client initialized
27-08-2022 10:46:52.557 Program initialized
27-08-2022 10:47:59.761 Writing config...
27-08-2022 10:48:00.062 Writing config: OK
```

**Comment by GreenZapperZ** [comment]
Steps to reproduce this issue (At least in VMPotato):

1. Launch Voicemeeter
2. Set a slider to anything above 0dB (Even 0.01dB works)
3. Launch VoicemeeterFancyOSD
4. Increase or decrease the dB of that slider by any amount
5. The number in the OSD should not be visible now and the slider should remain red despite lowering the dB below 0

I'm able to reproduce this issue 100% of the time by following these steps.
Hope this helps at all.

**Comment by A-tG** [comment]
Should be fixed in v1.1.0.6
